core/templates/pages/exploration-editor-page/editor-tab/state-param-changes-editor/state-param-changes-editor.component.spec.ts

```diff
@@ -16,37 +16,39 @@
  * @fileoverview Unit tests for stateParamChangesEditor component.
  */
 
-import { TestBed } from '@angular/core/testing';
+import { NO_ERRORS_SCHEMA } from '@angular/core';
+import { ComponentFixture, async, TestBed } from '@angular/core/testing';
+
+import { StateParamChangesEditorComponent } from
+  // eslint-disable-next-line max-len
+  'pages/exploration-editor-page/editor-tab/state-param-changes-editor/state-param-changes-editor.component';
 import { StateParamChangesService } from
   // eslint-disable-next-line max-len
   'components/state-editor/state-editor-properties-services/state-param-changes.service';
 
-require(
-  'pages/exploration-editor-page/editor-tab/state-param-changes-editor/' +
-  'state-param-changes-editor.component.ts');
-
-describe('State Param Changes Editor directive', function() {
-  var $scope = null;
-  var stateParamChangesService = null;
-
-  beforeEach(angular.mock.module('oppia'));
+let component: StateParamChangesEditorComponent;
+let fixture: ComponentFixture<StateParamChangesEditorComponent>;
+describe('State Param Changes Editor directive', () => {
+  let stateParamChangesService: StateParamChangesService = null;
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [StateParamChangesEditorComponent],
+      providers: [
+        StateParamChangesService
+      ],
```

Comment:
This might not be required. Can you remove once and check.

Reply:
Okay!

Reply:
But wont this be required to pass the `StateParamChangesService` to the constructor of `StateParamChangesEditorComponent`

Reply:
Done!